### PR TITLE
[DO NOT MERGE] - Review app infinite disco

### PR DESCRIPTION
**Pull request by egdbear**
The type of this PR is: **Feat**

This PR solves [PROJECT-XX]

### Description
https://infinite-disco.artsy.net/infinite-discovery

This PR is the POC of the Elastic Search recommendations. 

Screenshot 2024-11-21 at 11 32 38 AM
Screenshot 2024-11-21 at 11 32 48 AM

### CODE REVIEW

File: src/Apps/InfiniteDiscovery/InfiniteDiscoveryApp.tsx [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+      },
+    },
+    collapse: {
+      field: "artist_id",
+    },
+    aggregations: {
+      uniq_artists: {
+        terms: {
+          field: "artist_id",
+          size: 1,
```
Comment (cavvia) [inline src/Apps/InfiniteDiscovery/InfiniteDiscoveryApp.tsx:67]: I think on weaviate we allowed multiple works per artist in a batch... wondering whether this should be set at 2/3 to allow for a little more volume, worth testing I guess.

Comment (egdbear) [reply src/Apps/InfiniteDiscovery/InfiniteDiscoveryApp.tsx:67]: It is the product requirement to have 1 artist/artwork per batch
https://www.notion.so/artsy/Infinite-Discovery-Qualitative-QA-12fcab0764a0805c921bfe0146881f9c?pvs=4#13fcab0764a0804b96eae9d310b58c2d

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/e2e` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 92b26c287cb5fdddd6cd4bc36c56da7593c341b2
</p>

File: package.json [hunk 2]
```diff
@@ -147,6 +147,7 @@
     "lodash": "^4.17.21",
     "luxon": "^3.2.1",
     "map-cursor-to-max": "^1.0.0",
+    "mathjs": "^14.0.0",
```
Comment (egdbear) [inline package.json:150]: this will be removed once we set everything on the metaphysics, and potentially later on backend

**Comment by egdbear** [comment]
Closing in favor of https://github.com/artsy/force/pull/14997

**Pull request closed without merging**